eng/refreshManifests.ps1

```diff
@@ -0,0 +1,2 @@
+..\build.cmd
```

Comment:
Why does it build first?

Reply:
To make sure that everything is up to date because the loop uses --no-build (to speed things up).

Reply:
This shouldn't run build.cmd. Let users run that.